src/services/pieceService.js

```diff
@@ -0,0 +1,27 @@
+export const Pieces = [
```

Comment:
You're using this constant here and in boardService. This should go into that global const file - not sure where that went.